spec/system/distribution_system_spec.rb

```diff
@@ -395,6 +395,18 @@
       expect(@request.reload).to be_status_fulfilled
     end
 
+    it "copy over relevent request information into the distrubution" do
```

Comment:
"relevant" :slightly\_smiling\_face:

Reply:
Also, this doesn't need to be a request test since this is unrelated to the view->controller logic. It can be part of the test for the `copy_from_request` method on the distribution model. There isn't a test for this now, but it's probably a good idea to add one, even if it's bare bones, rather than covering this in a request test.

Reply:
Yes! That is a much better place, I put a test together! Let me know what you think